giraffa-core/src/test/java/org/apache/giraffa/GiraffaTestUtils.java

```diff
@@ -105,21 +106,6 @@ public static void enableMultipleUsers() {
     }
   }
 
-  /**
-   * Returns an INodeManager that is incapable of opening and closing multiple
-   * times because it has no coprocessor environment bound to it.
-   * @throws IOException
-   */
-  public static INodeManager getNodeManager(GiraffaConfiguration conf,
-                                            Connection connection,
-                                            RowKeyFactory<?> keyFactory)
-      throws IOException {
-    TableName tableName =
-        TableName.valueOf(getGiraffaTableName(conf));
-    Table table = connection.getTable(tableName);
-    return new INodeManager(keyFactory, table);
-  }
-
```

Comment:
I am +1 for these test changes. Should we get them in ASAP as they are not necessarily part of your work?